src/components/sidebar/Sidebar.tsx

```diff
@@ -35,6 +36,12 @@ interface ILinelistState {
 @inject('sidebarStore', 'routeStore', 'searchStore')
 @observer
 class Sidebar extends React.Component<ISidebarProps, ILinelistState> {
+
+    private renderRoutesView = () => {
+        const search = this.props.history.location.search;
+        return search === '' ?  <Redirect to='/' /> : <RoutesView />;
+    }
```

Comment:
The history things could be skipped if we utilize the navigator for this:
```
private renderRoutesView = () => {
    return !navigator.getQueryParam(QueryParams.routes)
        ? <Redirect to='/' /> : <RoutesView />;
}
```

Reply:
I would had put this logic inside the route view, and there just tell the navigator to navigate to '/'.